src/main/game/Language/Italian.java

```diff
@@ -0,0 +1,39 @@
+package main.game.Language;
+
+import main.game.Moves.Moves;
+
+import java.util.HashMap;
+
+import static main.game.Moves.Moves.*;
+
+public class Italian implements Language {
+
+    @Override
+    public String promptForMove() {
+        return "Scegli una mossa: ‘sasso’, ‘carta’ o ‘forbici’: ";
+    }
+
+    @Override
+    public String announceWinner(String convertedWinningMove) {
+        if (convertedWinningMove.contains("patta")) {
+            return "E'patta!";
+        } else {
+            return String.format("%s vince!", convertedWinningMove);
+        }
+    }
+
+    @Override
+    public String promptForGameMode() {
+        return "Inserisci '1' per Umani contro Umani\n" +
+                "Inserisci '2' per Uomo contro Computer";
+    }
+
+    @Override
+    public Moves translateToEnglish(String move) {
+        HashMap<Object, Object> translations = new HashMap<>();
```

Comment:
The `Object` in here denotes the _type_ of they key and the _type_ of value in the map.
As you are currently declaring your map to hold "Objects as keys for other Object values", you can pretty much stick anything in there, but also don't know what you are getting out. It could be `String`, but it could also be a `Tortoise` or a `Boolean`. Hence the need to typecase `(Moves)` in line 37.

If you change it to `Map<String, Moves>` the compiler (and IntelliJ) will know that the keys can only be `String` and that you will always get a `Moves` out. Anything else will be a compiler error. That means you won't need to do a typecast. And you get better guarantees about the correctness of your code.

Reply:
Yes! This was an oversight on my part. Thank you @felipesere - I agree with all of the above and will change it.